src/ws-dropdown/ws-dropdown.js

```diff
@@ -140,11 +140,11 @@ export class WSDropdown extends Component {
    * @returns {String}
    */
   getTextFromValue(value) {
-    let text = this.state.text;
+    let text = (this.state) ? this.state.text : '';
```

Comment:
See my changes in the other function. getTextFromValue is now called from createState in order to set the text value. That's where it would throw an error.